redhat-cop/containers-quickstarts
# Feature/jenkins slave zap

#### What is this PR About?
Addresses #100. Adds the Zed Attack Proxy Jenkins slave for security scanning. 

#### How do we test this?
See the readme for the commands to run in your pipeline. 

cc: @redhat-cop/containerize-it
